# HG changeset patch
# Parent d4d706483e1378477430197d3366a6e4d6dbb9cd
tabview is part of Firefox and not GRE

diff --git a/browser/base/content/tabview/tabview.js b/browser/base/content/tabview/tabview.js
--- a/browser/base/content/tabview/tabview.js
+++ b/browser/base/content/tabview/tabview.js
@@ -1,15 +1,15 @@
 const Cc = Components.classes;
 const Ci = Components.interfaces;
 const Cu = Components.utils;
 
-Cu.import("resource://gre/modules/tabview/AllTabs.jsm");
-Cu.import("resource://gre/modules/tabview/groups.jsm");
-Cu.import("resource://gre/modules/tabview/utils.jsm");
+Cu.import("resource:///modules/tabview/AllTabs.jsm");
+Cu.import("resource:///modules/tabview/groups.jsm");
+Cu.import("resource:///modules/tabview/utils.jsm");
 Cu.import("resource://gre/modules/Services.jsm");
 Cu.import("resource://gre/modules/XPCOMUtils.jsm");
 
 XPCOMUtils.defineLazyGetter(this, "gWindow", function() {
   return window.QueryInterface(Ci.nsIInterfaceRequestor).
     getInterface(Ci.nsIWebNavigation).
     QueryInterface(Ci.nsIDocShell).
     chromeEventHandler.ownerDocument.defaultView;
